src/msg/async/AsyncMessenger.cc

```diff
@@ -106,6 +106,7 @@ int Processor::bind(const entity_addr_t &bind_addr, const set<int>& avoid_ports)
   }
   net.set_close_on_exec(listen_sd);
   net.set_socket_options(listen_sd);
+  net.set_socket_options(listen_sd, msgr->cct->_conf->ms_tcp_nodelay, msgr->cct->_conf->ms_tcp_rcvbuf);
```

Comment:
and call `net.set_socket_options()` twice? might need to squash this commit into the previous one, to make sure every commit is compilable and atomic.